Ginger/GingerCore/ALM/JIRA/Bll/JiraConnectManager.cs

```diff
@@ -48,15 +48,19 @@ public bool SetJiraProjectFullDetails()
         {
             GetJiraDomainProjects();
             List<ProjectArea> currentProjects = jiraDomainsProjectsDataList.Where(x => x.DomainName.Equals(ALMCore.DefaultAlmConfig.ALMDomain)).Select(prjs => prjs.Projects).FirstOrDefault();
-            IProjectDefinitions selectedProj = currentProjects.Where(prj => prj.Prefix.Equals(ALMCore.DefaultAlmConfig.ALMProjectKey)).FirstOrDefault();
+            // olego 9/29/20 to be sure that nothing not impacted by change
+            // IProjectDefinitions selectedProj = currentProjects.Where(prj => prj.Prefix.Equals(ALMCore.DefaultAlmConfig.ALMProjectKey)).FirstOrDefault();
+            IProjectDefinitions selectedProj = currentProjects.Where(prj => prj.ProjectId.ToString() == ALMCore.DefaultAlmConfig.ALMProjectKey.ToString()).FirstOrDefault();
             if (selectedProj != null)
             {
                 //Save selected project details
                 connectedProjectDefenition = selectedProj;
                 ALMCore.DefaultAlmConfig.ALMProjectName = selectedProj.ProjectName;
-                ALMCore.DefaultAlmConfig.ALMProjectKey = selectedProj.Prefix;
+                // ALMCore.DefaultAlmConfig.ALMProjectKey = selectedProj.Prefix;
+                ALMCore.DefaultAlmConfig.ALMProjectKey = selectedProj.ProjectId.ToString();    // olego 9/29/20
                 JiraCore.ALMProjectGuid = selectedProj.Guid;
-                JiraCore.ALMProjectGroupName = selectedProj.Prefix;
+                // JiraCore.ALMProjectGroupName = selectedProj.Prefix;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)